src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+
+namespace NuGet.VisualStudio.Telemetry.PowerShell
```

Comment:
The namespace doesn't match the assembly

Reply:
@nkolev92 

Ok. Corrected. Previously I checked nearby other files in same project and took it.

https://github.com/NuGet/NuGet.Client/blob/59792e4733ced999af222a8ba7adb7a4ee8f5a08/src/NuGet.Clients/NuGet.VisualStudio.Common/UserInterfaceService/INuGetUILogger.cs#L6

Reply:
The namespace should be the assembly name. 

The INuGetUILogger type was likely moved recently from some other location *to* the NuGet.VisualStudio.Common.

Reply:
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio.Common/INuGetErrorList.cs

I will add that to the guidelines as well.